### PR TITLE
User onboarding

### PR DESCRIPTION
**Pull request by akanksha003**
So far, we have a basic tutorial for user onboarding set up. The tutorial gives a guided tour to the user when they first create an account with Stonks, so they can familiarize themselves with all the features available to them. 

Note: right now, the tutorial plays anytime the user refreshes the page. That behavior will be changed so that the user receives the tutorial only once when they log in, and then they can choose to replay the tutorial whenever they like by pressing the '?' icon in the nav bar.

### CODE REVIEW

File: client/src/pages/myProfile.jsx [hunk 1]
```diff
@@ -54,22 +55,20 @@ class Profile extends Component{
...
         <Col md={1}></Col>
         <Col md={10}>
-        {/*<h2 className="text-dark"> &nbsp; {user}'s Portfolio </h2>*/}
         <Portfolio items={items} titleLess={true} light isLoggedIn={!!user} enableClick={false}/>
         </Col></Row>
         <Row md={12}>
         <Col md={1}></Col>
         <Col md={10}>
-            <Tabs
+            <Tabs className={'tutorial2-step-2'}
```
Comment (willyspinner) [inline client/src/pages/myProfile.jsx:71]: This seems like it doesn't have a react-joyride tutorial. Is this intentional or a typo?

File: client/src/components/Chart.js [hunk 2]
```diff
@@ -34,10 +34,11 @@ class CandleStickChartWithCHMousePointer extends React.Component {
 		const xExtents = [start, end];
 
 		return (
+			<div className={'tutorial-step-2-b'}>
```
Comment (willyspinner) [inline client/src/components/Chart.js:37]: This one too, I don't see a `tutorial-step-2-b` anywhere in the joyride component?

**Review by willyspinner — CHANGES_REQUESTED** [review]
Nice work! Some things I'm a bit unclear about and then it's all good to go.

File: client/src/pages/myProfile.jsx (hunk 1 above)
Comment (akanksha003) [reply client/src/pages/myProfile.jsx:71]: hey! yes I forgot to remove those. This is from a previous version where the tutorial wasn't interactive.

File: client/src/components/Chart.js (hunk 2 above)
Comment (akanksha003) [reply client/src/components/Chart.js:37]: yea, thanks for pointing this out. I'll remove it now.

**Comment by akanksha003** [comment]
Good catch! I just removed those targets. Should be good now!

**Review by willyspinner — APPROVED** [review]
Go ahead and merge. Looks good to me

**Pull request merged**